translations/hu/get5.phrases.txt

```diff
@@ -340,8 +388,116 @@
     {
         "hu"            "[NEM ÁLL KÉSZEN]"
     }
+    "MapSelectionTeamChatOnly"
+    {
+        "hu"            "Csak csapat csevegőben írhatsz, pályaválasztás alatt."
+    }
+    "MapSelectionInvalidMap"
+    {
+        "hu"            "{1} nem érvényes pálya."
+    }
+    "MapSelectionPickMapHelp"
+    {
+        "hu"            "Használd a {1} <pálya>, hogy pályát kiválaszd."
```

Comment:
`Használd a {1} <pálya> parancsot, hogy pályát kiválaszd.`

Reply:
Elfogadva a javaslat. The suggestion is accepted